HISTORY.md

```diff
@@ -1,3 +1,7 @@
+## Unreleased
+
+* When using lein-environ, keywords with a `project` namespace are looked up in the Leiningen project map. E.g. `:project/version` returns the value of `:version` in the project map.
```

Comment:
Can you remove the changes to the changelog? That will be added later on release.